hack/build-images.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -euExo pipefail
+shopt -s inherit_errexit
+
+skip_header=true
+while IFS=, read -r context_dir image_tag from_tag
```

Comment:
would be good to sync these names with headers from csv

Reply:
done